kahuna/public/js/metadata-templates/metadata-templates.js

```diff
@@ -70,23 +97,43 @@ metadataTemplates.controller('MetadataTemplatesCtrl', [
 
   function applyTemplateToMetadata(templateMetadataFields) {
     if (templateMetadataFields && templateMetadataFields.length > 0) {
-      ctrl.metadata = angular.copy(ctrl.originalMetadata);
+      let metadata = angular.copy(ctrl.originalMetadata);
```

Comment:
[minor] this variable isn't reassigned (although is mutated) so it could be a `const`

Reply:
Good catch. Thank you 👍🏽